tasks/helpers.rb

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright:: Copyright (c) 2016 Chef Software Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Putting this method in a module so it is easier to test
+module RakeDependenciesTaskHelpers
+  def update_gemfile_from_stable(gemfile, stable_channel_name, gemfile_name, version_prefix="")
+    version = get_latest_version_for(stable_channel_name)
```

Comment:
This is calling with `stable_channel_name`, but the first argument of `get_latest_version` is `product_name`.